### DISCUSSION TITLE
Missing form values from the v-model

### DISCUSSION THREAD

**Discussion by gautham-8** [vueform/vueform#20] [Questions]
**Introduction**
We're trying to implement our custom uploader that works with FileElement and MultifileElement. Our implementation works with Vueform version 1.3.2 but we've noticed that with version 1.5.6, we're losing some properties from the file object that are needed for the rest of our code to work. We added a handler for the [`@updated` event here](https://stackblitz.com/edit/vitejs-vite-htwuht?file=src%2Fcomponents%2FMultifileModal.vue%3AL30) that logs out the value of v-model to the console. You can see that some of the properties like `name`, `size`, and `type` of the file object are lost:

<img width="573" alt="image" src="https://github.com/vueform/vueform/assets/83275001/7b7b7aa0-af82-454f-851a-e5b714fc4abd">

**How can you reproduce the issue?**

1. Click on `Open modal`
2. Upload a file
3. Inspect the last 2 console logs after the upload is complete.

What are we missing?

**Comment by szabolcsmucsi** [comment]
The problem is that you did not listen to the correct event, FileElement `@updated` will/might fire before the binary data is processed.

To have the file available the moment it is loaded, please use `@change`.
